Week1/homework/3-Log the number.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const numberx;
```

Comment:
Did you try to run this code? I think it will throw an error.

Reply:
u are right that different between const and let :) u can check it again